src/Execution/ExecutionContextBuilder.php

```diff
@@ -67,12 +69,25 @@ public function buildContext(
             throw new ExecutionException('Must provide an operation.');
         }
 
+        $variableValues = [];
+
+        /** @var OperationDefinitionNode $operation */
+        if ($operation) {
+            $coercedVariableValues = (new ValuesHelper())->coerceVariableValues(
```

Comment:
Use the associated function in `values.php` so that the helper can be overridden in the container.